perception/autoware_detected_object_validation/schema/object_lanelet_filter.schema.json

```diff
@@ -95,9 +95,42 @@
                 }
               },
               "required": ["enabled", "velocity_yaw_threshold", "object_speed_threshold"]
+            },
+            "debug": {
+              "type": "boolean",
+              "default": false,
+              "description": "If true, debug information is enabled."
+            },
+            "lanelet_extra_margin": {
+              "type": "number",
+              "default": 0.0,
+              "description": "Extra margin added to the lanelet boundaries."
+            },
+            "use_height_threshold": {
+              "type": "boolean",
+              "default": false,
+              "description": "If true, height thresholds are used to filter objects."
+            },
+            "max_height_threshold": {
+              "type": "number",
+              "default": 10.0,
+              "description": "Maximum height threshold for filtering objects (in meters)."
+            },
+            "min_height_threshold": {
+              "type": "number",
+              "default": -1.0,
+              "description": "Minimum height threshold for filtering objects (in meters)."
             }
```

Comment:
@YoshiRi 
Please add description which frame_id is based for these parameters?
I think it should be ` map/lanelet_frame_id_` as there is a transformation to `lanelet_frame_id_`  at 
https://github.com/autowarefoundation/autoware.universe/blob/f83aa01a6c80986e43f414938b8f77471be77e1b/perception/autoware_detected_object_validation/src/lanelet_filter/lanelet_filter.cpp#L149-L154

Reply:
Thanks! Fixed in the latest test.